vb-fe/src/components/loading/Loading.tsx

```diff
@@ -1,30 +1,35 @@
 import React, { FunctionComponent } from "react";
 
-import { Modal, ModalOverlay, ModalContent, ModalHeader, ModalBody, Icon } from "@chakra-ui/react";
+import { Modal, ModalOverlay, ModalContent, ModalHeader, ModalBody, Icon, Text, Box } from "@chakra-ui/react";
 
 import { ReactComponent as LoadingIcon } from "assets/icons/loading.svg";
 
 import "components/loading/Loading.scss";
+import { useSelector } from "react-redux";
+import { State } from "_store/rootReducer";
 
 interface Props {
   show: boolean;
 }
 
 export const Loading: FunctionComponent<Props> = ({ show }) => {
-  const onClose = () => {
-    console.log("Loading closed");
-  };
+  const loadingText = useSelector((state: State) => state.system.loadingText);
 
   return (
     <div>
-      <Modal isOpen={show} onClose={onClose} size="xs" autoFocus={false} closeOnOverlayClick={false} isCentered>
+      <Modal isOpen={show} onClose={Function} size="xs" autoFocus={false} closeOnOverlayClick={false} isCentered>
         <ModalOverlay />
         <ModalContent className="bg-secondary text-white">
-          <ModalHeader className="text-center">Loading...</ModalHeader>
+          <ModalHeader className="text-center">Getting reading to vibe!</ModalHeader>
```

Comment:
Lol 'Getting ready to vibe!'

I love it 😆

Reply:
lmao does it sound too corny? LOL